src/aap_eda/api/views/eda_credential.py

```diff
@@ -277,3 +279,46 @@ def destroy(self, request, *args, **kwargs):
             )
         self.perform_destroy(eda_credential)
         return Response(status=status.HTTP_204_NO_CONTENT)
+
+    @extend_schema(
+        description="Copy an EDA credential.",
+        request=None,
+        responses={
+            status.HTTP_201_CREATED: OpenApiResponse(
+                serializers.EdaCredentialSerializer,
+                description="Return the copied EDA credential.",
+            ),
+            status.HTTP_403_FORBIDDEN: OpenApiResponse(
```

Comment:
I assume a specific test for this wasn't added based on RBAC enforcement for `self.get_object()` and the serializer create? 
Given our usage of `self.get_object()` in other locations where we don't list a 403 as a possible response are we being inconsistent?

The above was my original comment but this popped into my head:
Can we have a situation where a user could get an object but not be able to create a new one?

Reply:
I'm sure we are being inconsistent in other places where we don't list 403 as one of possible responses. But this is mainly for API schema documentation, thus it has little to no effect on the functionality. 

As for the second use case you mentioned, it is possible and can easily happen when a user has object-level role on one credential but has no org-level credential role because this copy endpoint requires `add_edacredential` permission to succeed.

Reply:
> I'm sure we are being inconsistent in other places where we don't list 403 as one of possible responses. But this is mainly for API schema documentation, thus it has little to no effect on the functionality.

Wasn't concerned about functionality but the consistency of what we're doing and presenting.  We should be consistent.  The reasonable (to me) way of doing that would be to only include a 403 response in the schema if it is a response EDA itself can create and not for transitive responses.  

Providing consistency would be a separate effort from this change, though you could remove the occurrence here.  Regardless, I'm still going to approve this change. 😄

Reply:
Removed the 403 response as per your feedback to stay consistent.